color.go

```diff
@@ -18,8 +18,7 @@ var (
 	// or not. This is a global option and affects all colors. For more control
 	// over each color block use the methods DisableColor() individually.
 	NoColor = os.Getenv("TERM") == "dumb" ||
-		!isatty.IsTerminal(os.Stdout.Fd()) ||
-		!isatty.IsCygwinTerminal(os.Stdout.Fd())
+		(!isatty.IsTerminal(os.Stdout.Fd()) || !isatty.IsCygwinTerminal(os.Stdout.Fd()))
```

Comment:
Still wrong, I think.
If both IsTerminal() and IsCygwinTerminal() return false, color cannot be used.
At least one of them returns true, color can be used.